language/lang.go

```diff
@@ -189,4 +189,11 @@ type GenerateResult struct {
 	// correspond. These values are passed to Resolve after merge. The type
 	// is opaque since different languages may use different representations.
 	Imports []interface{}
+
+	// RelsToIndex is a list of additional directories to index for dependency
```

Comment:
Does `a/b` indicate `a/b/*` should also be indexed? So including the root directory causes a full indexing of the repo?

Reply:
Should `a/b/c` index `a/b` even if `a/b/c` does not exist (but its parent directories do)?

Reply:
> Does a/b indicate a/b/* should also be indexed? So including the root directory causes a full indexing of the repo?

Good question. No, only `a/b` should be indexed, not subdirectories. I've updated the doc comment to be more explicit about this and disabled recursion in `walk.Walk2`.

> Should a/b/c index a/b even if a/b/c does not exist (but its parent directories do)?

I'm not sure. Initially I thought no, but your comment below about avoiding the stat made me think it's simpler to index parents. Changed to that.